lib/spike-collisions.js

```diff
@@ -0,0 +1,39 @@
+export default class SpikeCollisions {
+  constructor(players, flags, spikes) {
+    this.players = players
+    this.flags = flags
+    this.spikes = spikes 
+    this.touchRadius = 50
+  }
+
+  spikeCollision() {
+    let radius = this.touchRadius
+    this.players.forEach(player => {
+      this.spikes.forEach(spike => {
+        if (player.x <= spike.x + radius && player.x >= spike.x && player.y <= spike.y + radius && player.y >= spike.y) {
```

Comment:
I recycled some code here from the other collisions classes. There was an error with the last check. I think I was checking `player.y >= spike.y - radius` , but then released it should be be `player.y >= spike.y`. Maybe we were using that for checking wall collisions.